### ISSUE TITLE
Certain Invisible Item Drops

### ISSUE THREAD

**Issue by GentlemanCheesy** [retrooper/packetevents#1016]
**Describe the bug**
Certain items containing specific item components will be invisible while dropped from your inventory.

**Software brand**
Paper 1.21.1

**Plugins**
![image](https://github.com/user-attachments/assets/5dfa8ad7-349b-4f25-a397-88288789fcaa)

**How To Reproduce**
1. Obtain an item with item components similar to the item in question.
2. Drop it on the ground.
3. Observe it being invisible.
4. Try to pick it up to determine it was dropped.

**Expected behavior**
Item in question should be visible when dropped.

**Screenshots**
N/A

**Additional context**
Server Latest.log on pastebin: https://pastebin.com/La90Qa8c
"/paper dumpitem" output:
minecraft:paper[jukebox_playable={song: "redux:moyai"},custom_model_data=9000067,max_stack_size=1,custom_name='{"extra":[{"bold":false,"italic":false,"obfuscated":false,"strikethrough":false,"text":"Music Disc","underlined":false}],"text":""}',custom_data={CheesyCraft: "MoyaiManor"},rarity="rare"]

**Comment by GentlemanCheesy** [comment]
Note that this might be related to (https://github.com/retrooper/packetevents/issues/1011)

**Comment by Vinrey123** [comment]
Yes, most likely, the component of the effect contained in the food is to blame for this, well, for me personally.

**Comment by GentlemanCheesy** [comment]
This issue is still present with the "jukebox_playable" item component from the "/paper dumpitem" above.
Tested on dev builds:
![image](https://github.com/user-attachments/assets/37770365-d19d-4ed4-b053-63295e2d1164)

**Comment by booky10** [comment]
Are there any errors shown in the console?

**Comment by GentlemanCheesy** [comment]
Here is a snippet of the logs immediately after dropping said item on the ground, which spams the console.
https://pastebin.com/hxXJkhP3

**Comment by NoJokeFNA** [comment]
Does it still happen on the latest version? @GentlemanCheesy

**Comment by GentlemanCheesy** [comment]
Cannot replicate this issue on current version of packet-events. Same item as the original works fine.

**Comment by NoJokeFNA** [comment]
Great! :)
@booky10 can be closed
